pre-cache/precache.sh

```diff
@@ -1,12 +1,22 @@
-#!/bin/bash
+#!/usr/bin/bash
 
 set -e
 
+# Setup the environment for running podman.
+mkdir /host/dev
+mknod -m 0666 /host/dev/null c 1 3
+mkdir /host/dev/shm
+/host/usr/bin/mount -t tmpfs -o rw,nosuid,nodev,noexec,relatime,size=2G tmpfs /host/dev/shm
```

Comment:
You should not specify the tmpfs size explicitly. It defaults to half the size of the RAM, and can be viewed using `df`. See https://www.kernel.org/doc/html/latest/filesystems/tmpfs.html for details. On a regular system, `systemd` seems to do the same thing for `/dev/shm` (not specify the size explicitly).